src/Assignments/SimpleProblemPage.tsx

```diff
@@ -43,13 +44,16 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
     };
 
     const params = useParams<SimpleProblemPageLocationParams>();
+    const {getCurrentQueryStrings, updateRoute} = useQuerystringHelper();
+    const urlProblemIdQS = getCurrentQueryStrings()?.['problemId'];
```

Comment:
Why do you need the brackets and quotes?

Reply:
That's the way getCurrentQueryStrings is structured. Everything in the URL is a string, and may or may not be there.